webapp/src/components/App/Settings/Tokens/dialog/utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import type { RoleType } from "@/types/types";
+
+export interface TokenFormDefaultValues {
+  name: string;
+  permissions: Array<{
+    group: {
+      id: string;
+      name: string;
+    };
+    type: RoleType;
+  }>;
+}
```

Comment:
its technically correct, but better to extract those kind of nested types for better readability 
```ts
export interface TokenPermission {
  group: {
    id: string;
    name: string;
  };
  type: RoleType;
}

export interface TokenFormDefaultValues {
  name: string;
  permissions: TokenPermission[];
}
```